packages/contracts/test/deploy/updateTo1_3_0.ts

```diff
@@ -1,16 +1,29 @@
 import {expect} from 'chai';
 
 import {deployments} from 'hardhat';
-import {initForkAndFixture} from '../test-utils/fixture';
-import {v1_0_0_active_contracts} from '@aragon/osx-versions';
+import {
+  ForkOsxVersion,
+  initForkForOsxVersion,
+  initializeDeploymentFixture,
+} from '../test-utils/fixture';
+import {activeContractsList as v1_2_0_activeContracts} from '@aragon/osx-ethers-v1.2.0';
 
 const enableTest = process.env.TEST_UPDATE_DEPLOY_SCRIPT !== undefined;
 const network = 'mainnet';
 
 if (enableTest) {
   describe('update/to_v1.3.0', function () {
     before(async () => {
-      await initForkAndFixture(network, 'v1_3_0', 'v1_0_0');
+      const previousOsxVersion: ForkOsxVersion = {
+        version: 'v1.0.1',
```

Comment:
I'm lacking a lot of context about this,  but why is version `v1.0.1` and then active contracts are `v1_2_0`